ui-common/src/components/TrafficViewer/EntriesList.tsx

```diff
@@ -156,7 +159,6 @@ export const EntriesList: React.FC<EntriesListProps> = ({
         case "queryMetadata":
           setTruncatedTimestamp(message.data.truncatedTimestamp);
           setQueriedTotal(message.data.total);
-          setLeftOffTop(leftOffState => leftOffState === "" ? message.data.leftOff : leftOffState);
```

Comment:
Left off value should come from the metadata stream. Why do we remove this `setLeftOffTop` call?